src/lib/y2network/sysconfig/config_writer.rb

```diff
@@ -36,33 +36,40 @@ module Sysconfig
     class ConfigWriter
       include Yast::Logger
 
+      # @return [Array<Symbol>] The different sections handled by the writer
+      SECTIONS = [:routing, :drivers, :interfaces, :connections, :dns, :hostname].freeze
+
       # Writes the configuration into YaST network related modules
       #
       # @param config     [Y2Network::Config] Configuration to write
       # @param old_config [Y2Network::Config] Old configuration
-      def write(config, old_config = nil)
+      def write(config, old_config = nil, sections: :all)
+        sections = SECTIONS if sections == :all
         log.info "Writing configuration: #{config.inspect}"
         log.info "Old configuration: #{old_config.inspect}"
-        write_ip_forwarding(config.routing) if config.routing
-        write_interface_changes(config, old_config)
-
-        # update /etc/sysconfig/network/routes file
-        file = routes_file_for(nil)
-        file.routes = find_routes_for(nil, config.routing.routes)
-        file.save
+        write_interface_changes(config, old_config) if sections.include?(:interfaces)
```

Comment:
just document why it is needed also here beside below

Reply:
Checking it seems it is safe enough to be handled by the write_interfaces method, so, moving it.

Reply:
In fact, it is somehow a dependency between the interfaces and the configured routes. As it writes or removes the configured interfaces routes I will move to the `write_routing` method.